server/pbenchinacan/etc/pbench-server/pbench-server.cfg

```diff
@@ -28,17 +28,12 @@ port = 9200
 [database]
 uri = postgresql://pbenchcontainer:pbench@pbenchinacan:5432/pbenchcontainer
 
-# User authentication section to use when authorizing the user with an OIDC identity provider
+# User authentication section to use when authorizing the user with an OIDC
+# identity provider.
 [authentication]
 
 # URL of the OIDC auth server
-server_url = http://localhost:8090
-
-# [optional] URL of the OIDC auth server for pbench-server service.
-# It is possible that the OIDC auth server may not be accessible on the same URL
-# as specified above for the pbench-server service. If that is the case specify
-# the URL that pbench-server can use for authentication.
-internal_server_url = http://localhost:8080
```

Comment:
We apparently did not reference the `internal_server_url` anywhere.  So I remove it to avoid confusion.  If a future code change needs that, it should be a derived value and not another redundant setting.

Reply:
I just looked it up, and the `/api/vi/endpoints` interface is exporting the value of `server_url`; the value of `internal_server_url`, prior to your changes, was used to configure the Pbench Server's access to the OIDC server, with `server_url` as a fallback if `internal_server_url` was not provided.

This arrangement makes good sense if the Pbench Server is supposed to use an address different from the one which the client is supposed to use (which seems like it would be the case when the two servers are run in a pod).

So, before you just tear out this mechanism, have you established that in all cases the Pbench Server can access the OIDC server using the same address that the client uses?

Reply:
Yes.

Reply:
Okie....